python/docs/source/reference.rst

```diff
@@ -9,6 +9,19 @@ Manage characteristics
 ^^^^^^^^^^^^^^^^^^^^^^
 
 Recode/rename characteristic columns, merge HIE data, and fix characteristic
-errors.
+errors
 
-:doc:`vars_rename() <vars_rename>`
+:doc:`vars_rename() <vars_rename>` |nbsp|
+:doc:`vars_recode() <vars_recode>`
+
+Data
+----
+
+Dictionaries
+^^^^^^^^^^^^
+
+Lookups for numeric codes used in the assessment system
```

Comment:
**nitpick:** Also used for `vars_rename`, so not just for the numeric code lookup.

Reply:
Good point, clarified in 2a82c96791830b07b1460e7b66df6df3c728df03.